src/model/Model.hpp

```diff
@@ -126,6 +158,26 @@ namespace model {
    *  object which can be significantly faster than calling getOptionalUniqueModelObject<OutputControlFiles>(). */
     boost::optional<OutputControlFiles> outputControlFiles() const;
 
+    /** Get the OutputControlReportingTolerances object if there is one, this implementation uses a cached reference to the OutputControlReportingTolerances
+   *  object which can be significantly faster than calling getOptionalUniqueModelObject<OutputControlReportingTolerances>(). */
+    boost::optional<OutputControlReportingTolerances> outputControlReportingTolerances() const;
```

Comment:
Note: I had to delete the `Version` one. Because it was shadowing the `IdfFile::version()` https://github.com/NREL/OpenStudio/blob/7b76d3ae5ab65aea88e6c6e5cd0f4dda4f698600/src/utilities/idf/IdfFile.hpp#L87-L88

That's why the VersionTranslator suddenly needed adjustments FYI @joseph-robertson